trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core.tests/src/org/eclipse/tracecompass/incubator/trace/server/jersey/rest/core/tests/services/ConfigurationManagerServiceTest.java

```diff
@@ -306,8 +362,29 @@ private static Response createConfig(String path, boolean isCorrectType) {
         if (path != null) {
             parameters.put(PATH, path);
         }
+        ObjectMapper mapper = new ObjectMapper();
         return endpoint.request(MediaType.APPLICATION_JSON)
-                .post(Entity.json(new QueryParameters(parameters, Collections.emptyList())));
+                .post(Entity.json(new ConfigurationQueryParameters(mapper.valueToTree(parameters))));
+    }
+
+    private static Response createJsonConfig(String jsonFileName) throws URISyntaxException, IOException {
+        String typeId = CONFIG_WITH_SCHEMA_ANALYSIS_TYPE_ID;
+        WebTarget endpoint = getApplicationEndpoint()
+                .path(CONFIG_PATH)
+                .path(TYPES_PATH)
+                .path(typeId)
+                .path(CONFIG_INSTANCES_PATH);
+
+        Bundle bundle = Platform.getBundle("org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core.tests");
+        IPath defaultPath = new org.eclipse.core.runtime.Path("config/custom-execution-analysis.json");  //$NON-NLS-1$
```

Comment:
I noticed that the json file name (without the path "config/") is defined at the top of the file, and passed to this function, but still a hardcoded value is used here?

Reply:
Oh, there is a typo in the file name, as defined at the top of the file, so it would not have been found if used. See somment up there.

Reply:
Good catch. I'll use the constant and fix the typo.